h2o-r/h2o-package/R/models.R

```diff
@@ -918,6 +918,55 @@ predict_contributions.H2OModel <- function(object, newdata, output_format = c("o
 #' @export
 h2o.predict_contributions <- predict_contributions.H2OModel
 
+#' Output row to tree assignment for the model and provided training data.
+#'
+#' Output is frame of size nrow = nrow(original_training_data) and ncol = number_of_trees_in_model+1 in format: 
+#'     row_id    tree_1    tree_2    tree_3
+#'          0         0         1         1
+#'          1         1         1         1
+#'          2         1         0         0
+#'          3         1         1         0
+#'          4         0         1         1
+#'          5         1         1         1
+#'          6         1         0         0
+#'          7         0         1         0
+#'          8         0         1         1
+#'          9         1         0         0
+#' 
+#' Where 1 in the tree_{number} cols means row is used in the tree and 0 means that row is not used.
+#' The structure of the output depends on sample_rate or sample_size parameter setup.
+#'
+#' @param object a fitted \linkS4class{H2OModel} object
+#' @param original_training_data An H2OFrame object that was used for model training. Currently there is no validation of the input.
+#' @examples
+#' \dontrun{
+#' library(h2o)
+#' h2o.init()
+#' prostate_path <- system.file("extdata", "prostate.csv", package = "h2o")
+#' prostate <- h2o.uploadFile(path = prostate_path)
+#' prostate_gbm <- h2o.gbm(4:9, "AGE", prostate, sample_rate = 0.6)
```

Comment:
What is meant by `4:9`?

Reply:
use only columns 4,5,6,7,8 for prediction

Reply:
ok, thanks!